libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts

```diff
@@ -24,43 +24,48 @@ import { eoFaqRoutePath } from '@energinet-datahub/eo/faq/routing';
     `
       :host {
         display: block;
-        width: 800px; // Magic number by designer
+        width: 1040px; // Magic number by designer
       }
 
-      h3 {
+      .faq-link a {
+        color: var(--watt-color-primary);
+        text-decoration: none;
+      }
+
+      h2 {
         margin-top: var(--watt-space-xl);
         margin-bottom: var(--watt-space-s);
       }
     `,
   ],
   template: `
-    <div class="watt-space-stack-m">
-      <a href="${eoFaqRoutePath}#what-is-energy-origin"
-        >What is Energy Origin?</a
-      >
+    <div class="watt-space-stack-m faq-link">
+      <a href="${eoFaqRoutePath}#what-is-energy-origin">
+        What is Energy Origin?
+      </a>
     </div>
-    <div class="watt-space-stack-m">
-      <a href="${eoFaqRoutePath}#who-can-access-the-platform"
-        >Who can access the platform?</a
-      >
+    <div class="watt-space-stack-m faq-link">
+      <a href="${eoFaqRoutePath}#who-can-access-the-platform">
+        Who can access the platform?
+      </a>
     </div>
-    <div class="watt-space-stack-m">
-      <a href="${eoFaqRoutePath}#where-does-the-data-come-from"
-        >Where does the data come from?</a
-      >
+    <div class="watt-space-stack-m faq-link">
+      <a href="${eoFaqRoutePath}#where-does-the-data-come-from">
+        Where does the data come from?
+      </a>
     </div>
-    <div class="watt-space-stack-m">
-      <a href="${eoFaqRoutePath}#how-can-i-influence-the-development"
-        >How can I influence the development?</a
-      >
+    <div class="watt-space-stack-m faq-link">
+      <a href="${eoFaqRoutePath}#how-can-i-influence-the-development">
+        How can I influence the development?
+      </a>
     </div>
-    <div class="watt-space-stack-m">
-      <a href="${eoFaqRoutePath}#where-can-i-read-more"
-        >Where can I read more?</a
-      >
+    <div class="watt-space-stack-m faq-link">
+      <a href="${eoFaqRoutePath}#where-can-i-read-more">
+        Where can I read more?
+      </a>
     </div>
 
```

Comment:
question:

This (h2 vs h3) was previously agreed and approved with designer, has the design changed?

Reply:
Checking with UX again. Will get back to you :)

Reply:
UX changed their design to H3 now, so the code has been changed to reflect this now :)